examples/todo-list/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fragstore-todolist",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [],
+  "main": "src/index.js",
+  "dependencies": {
+    "@testing-library/dom": "8.10.1",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "12.1.2",
+    "@testing-library/user-event": "13.5.0",
+    "fragstore": "0.4.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-icons": "^4.3.1",
+    "react-scripts": "4.0.0",
+    "teaful": "^0.7.0"
```

Comment:
```suggestion
    "teaful": "link:../"
```

Reply:
A we talked about this, we keep a fixed version of teaful at the moment to test manually  this examples when version is updated to be sure all is working fine